x/app-router/types/parser.go

```diff
@@ -18,30 +18,33 @@ type ParsedReceiver struct {
 // need args: Creator, HostDenom, Amount
 func ParseReceiverData(receiverData string) (*ParsedReceiver, error) {
```

Comment:
As part of [ibc-go v5.1.0](https://github.com/cosmos/ibc-go/releases/tag/v5.1.0), an optional [memo field](https://github.com/cosmos/ibc-go/blob/v5.1.0/proto/ibc/applications/transfer/v2/packet.proto#L20) has been added to FungibleTokenPacketData.

Is it possible to check the packet for the memo field here, and if it exists, use that instead? In the future, we could migrate towards using the memo field exclusively.

Reply:
I think memo field check could be done on v5 upgrade. `FungibleTokenPacketData` does not have memo field right now.

Reply:
Done https://github.com/Stride-Labs/stride/pull/432/commits/eae019f7e87b79f7d68692cdfab917bf59cdf3cf